locations/spiders/gold_union_fr.py

```
@@ -0,0 +1,16 @@
+from typing import Iterable
+
+from locations.categories import Categories, apply_category
+from locations.items import Feature
+from locations.storefinders.stockist import StockistSpider
+
+
+class GoldUnionFRSpider(StockistSpider):
+    name = "gold_union_fr"
+    item_attributes = {"brand": "Gold Union", "brand_wikidata": "Q131622916"}
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)